src/SFA.DAS.DATA.MANAGEMENT/SFA.DAS.Data.Management.Database.sqlproj.user

```diff
@@ -0,0 +1,3 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+</Project>
```

Comment:
this looks like user profile file, may not be necessary in this PR. can you double check please.

Reply:
Yes, This is not required. Sarma, Can you remove this please. Thank You.